src/main/java/service/Bot.java

```diff
@@ -0,0 +1,334 @@
+package service;
+
+import org.telegram.telegrambots.bots.TelegramLongPollingBot;
+import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
+import org.telegram.telegrambots.meta.api.objects.Message;
+import org.telegram.telegrambots.meta.api.objects.Update;
+import org.telegram.telegrambots.meta.exceptions.TelegramApiException;
+import repository.steps.*;
+import repository.strings.Storage;
+
+public class Bot extends TelegramLongPollingBot {
+    private CountryOrigin countryOrigin = null;
+    private OwnersType ownersType = null;
+    private CarAge carAge = null;
+    private TypeOfEngine typeOfEngine = null;
+    private VolumeOfEngine volumeOfEngine = null;
+    private final CalculatorPassenger calculator;
+
+    //singleton Bot creation
+    private static class BotHolder {
+        private static final Bot bot = new Bot();
+    }
+
+    //bot's name
+    private static final String BOT_NAME = "Бесплатный калькулятор утильсбора РБ";
+
+    //private constructor to avoid wrong bot's creation
+    private Bot() {
+        calculator = CalculatorPassenger.getInstance();
+    }
+
+    //bot's token takes from the environmental variables
+    @Override
+    public String getBotToken() {
+        return System.getenv("custom_by_utilsbor_bot");
+    }
+
+    //method describes what to do after receiving message
+    @Override
+    public void onUpdateReceived(Update update) {
+        //if user send us smth
+        SendMessage sendMessage;
+        Message inputMessage = update.getMessage();
+        try {
+            if (update.hasMessage() && inputMessage.hasText()) {
+                sendMessage = checkInputMessage(inputMessage);
+            } else sendMessage = getSorryMessage(inputMessage);
```

Comment:
Better to use curl brackets even for a single line code.
```suggestion
            } else {
               sendMessage = getSorryMessage(inputMessage);
            }
```

Reply:
Fixed.